testdata/kconfig_config.c

```diff
@@ -0,0 +1,73 @@
+#include "common.h"
+
+char __license[] __section("license") = "MIT";
+
+// Highly inspired from libbpf test:
+// https://github.com/torvalds/linux/blob/6e98b09da931a00bf4e0477d0fa52748bf28fcce/tools/testing/selftests/bpf/progs/test_core_extern.c
+// The modifications are the following:
+// 1. Removed __weak as cilium/ebpf does not support it.
```

Comment:
Just to make sure we don't introduce behaviour that is incompatbile with adding __weak later: what does libbpf do when it finds a symbol that is marked weak but doesn't exist?